pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -324,6 +324,7 @@ type TidbClusterSpec struct {
 	StatefulSetUpdateStrategy apps.StatefulSetUpdateStrategyType `json:"statefulSetUpdateStrategy,omitempty"`
 
 	// PodManagementPolicy of TiDB cluster StatefulSets
+	// +kubebuilder:validation:Enum=OrderedReady;Parallel
```

Comment:
Does this work if this field is not set explicitly?

Reply:
@DanielZhangQD Do you mean we need a default value for this field? If so, what is the default value?